docs/releases/1.9.3.txt

```diff
@@ -37,3 +37,7 @@ Bugfixes
 
 * Reallowed dashes in top-level domain names of URLs checked by
   ``URLValidator`` to fix a regression in Django 1.8 (:ticket:`26204`).
+
+* Reverted a change in Django 1.9.2 that prevented abstract models with app
+  relative lazy relationships from being used as reusable blueprints
```

Comment:
Maybe you want to link to the section that describes the behavior as it's not self-explanatory what "reusable blueprints" means.